Automattic/wp-calypso
# Help Center: move odie logic into component

## Proposed Changes

* Move all Odie logic into `<HelpCenterOdie />`
* Rename all `elegible` to `eligible`

## Why are these changes being made?

The main routing component for the Help Center `HelpCenterContent` contains a significant amount of logic and functions that are only used to connect with Odie. There is currently already a Help Center component to display the Odie client. As the Help Center continues to grow I feel it may be best to keep logic organized to prevent technical debt and preserve readability.

## Testing Instructions

1. Open Calypso live link
2. Open Help Center and go to Wapuu.
3. Check for regressions.
